packages/roosterjs-editor-core/lib/coreApi/selectImage.ts

```diff
@@ -21,3 +30,38 @@ export const selectImage: SelectImage = (image: HTMLImageElement | null) => {
 
     return null;
 };
+
+const select = (core: EditorCore, image: HTMLImageElement) => {
+    const styleTagId = STYLE_ID + core.contentDiv.id;
+    const doc = core.contentDiv.ownerDocument;
+    let styleTag = doc.getElementById(styleTagId) as HTMLStyleElement;
+    if (!styleTag) {
+        styleTag = doc.createElement('style');
+        styleTag.id = styleTagId;
+        doc.head.appendChild(styleTag);
+    }
+
+    const borderCSS = buildBorderCSS(core, image.id);
+    styleTag.sheet?.insertRule(borderCSS);
+};
+
+const buildBorderCSS = (core: EditorCore, imageId: string): string => {
+    const borderColor = core.imageSelectionBorderColor || '#DB626C';
+    return (
+        '#' +
+        core.contentDiv.id +
+        ' #' +
+        imageId +
+        ' { margin: -2px; border: 2px solid' +
```

Comment:
question: do we need to add "!important" to the styles in case the image already has some conflict style?

(same for table selection)

Reply:
I noticed table selection already has "!important". I also added to image selection.

Reply:
If the inline style contains !important will have higher priority than the head style, I think in table selection I added a function to remove the !important in case it is present in the inline style.

Reply:
I abstract the code for removeImportant function and made a new one so we can use it in both selectImage and selectTable.